Resources/public/css/theme/views/languageswitcher.css

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-language-switcher-container {
+}
+
+.ez-language-switcher-container .ez-dropdown-list-indicator {
```

Comment:
the `.ez-language-switcher-container` element does not really belong to the language switcher so you should not style anything with it. Instead, you should use the automatically generated class on the view container which should be something like `.ez-view-languageswitcherview`